src/com/projectkorra/ProjectKorra/Commands.java

```diff
@@ -1218,10 +1218,8 @@ public void run() {
 						}
 						BendingPlayer bTarget = Methods.getBendingPlayer(target.getName());
 
-						if (bTarget.isPermaRemoved()) {
-							s.sendMessage(ChatColor.RED + "That player's bending was permanently removed.");
-							return true;
-						}
+						if (bTarget.isPermaRemoved()) bTarget.setPermaRemoved(false);
```

Comment:
I don't understand why you would setPermaRemoved() to false here. It should be done when the player is banned